src/ctl.rs

```diff
@@ -1,6 +1,6 @@
 use derive_more::{Constructor, Display as MDisplay};
 use std::fmt::Display as FDisplay;
-use strum::{Display as SDisplay, EnumProperty};
+// use strum::{Display as SDisplay, EnumProperty};
```

Comment:
If there is no usage of strum, wipe out this comment.

Reply:
👍